src/pages/home/HomePage.tsx

```diff
@@ -38,7 +46,10 @@ export default class HomePage extends React.Component<IHomePageProps, {}>
     public render()
     {
         return (
-       		<QueryContainer/>
+        	<FlexCol style={{height: '100%'}}>
+				<a style={{alignSelf: 'center'}} onClick={() => this.setState({ mode: this.state.mode == 2 ? 1 : 2 })}>Switch to {this.state.mode == 2 ? 'old' : 'new'} view</a>
```

Comment:
I don't have a good reason to use three. I know when three are necessary and I know in every case I use 2 it is ok to use 2. I never make errors with 3 versus 2 equals and typescript makes it even less necessary to use 3.